src/sentry/static/sentry/app/views/issueList/actions.tsx

```diff
@@ -646,6 +646,11 @@ class IssueListActions extends React.Component<Props, State> {
           <AssigneesLabel className="align-right hidden-xs hidden-sm">
             <ToolbarHeader>{t('Assignee')}</ToolbarHeader>
           </AssigneesLabel>
+          <Feature organization={organization} features={['organizations:inbox']}>
+            <ActionsLabel className="align-right hidden-xs hidden-sm">
```

Comment:
Since this is a styled component, can we remove `align-right` and apply to styled component instead

Reply:
👍